algosdk/testing/dryrun.py

```diff
@@ -42,6 +43,188 @@ class App:
     global_state: List[TealKeyValue] = None
 
 
+#### LIGHTWEIGHT ASSERTIONS FOR RE-USE ####
```

Comment:
Refactor `DryrunTestCaseMixin` so can have common assertions with `teal_blackbox.py`

Reply:
These changes have unit tests to back them and I've also added the new file `x/blackbox/dryrun_mixin_docs_test.py` for further confidence.